File: Reverse Integer.cs
public class Solution {
    public int Reverse(int x) {
        
        long reverse=0;
        for(int i=0;x!=0;i++){
            int temp=x%10;
            reverse=reverse*10+temp;
            x=x/10;
        }
        if(reverse >= -2147483648 && reverse <=  2147483647)
            return (int)reverse;      

        return 0;
    }
}

File: 7-reverse-integer/reverse-integer.py
class Solution:
    def reverse(self, x: int) -> int:
        isNegative = False
        
        if x < 0:
            isNegative = True
            x = abs(x)
        
        result=0
        while x>0:
            result=result*10+x%10
            x=x//10
        
        if result>2**31-1:
            return 0
        return result*-1 if isNegative else result